service/frontend/admin_handler.go

```diff
@@ -1860,6 +1860,79 @@ func (adh *AdminHandler) MergeDLQTasks(ctx context.Context, request *adminservic
 	}, nil
 }
 
+func (adh *AdminHandler) DescribeDLQJob(ctx context.Context, request *adminservice.DescribeDLQJobRequest) (*adminservice.DescribeDLQJobResponse, error) {
+	jt := adminservice.DLQJobToken{}
+	err := jt.Unmarshal([]byte(request.JobId))
+	if err != nil {
+		return nil, errInvalidDLQJobToken
```

Comment:
You should wrap this error so that we get the specifics on what made it invalid. For example: `fmt.Errorf("%w: %v", errInvalidDLQJobToken, err)`.